huaweicloud/services/ims/resource_huaweicloud_images_image.go

```diff
@@ -106,6 +113,11 @@ func ResourceImsImage() *schema.Resource {
 				ForceNew: true,
 				Computed: true,
 			},
+			"vault_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
it's better add `RequiredWith: []string{"instance_id"},` here

Reply:
done